src/pages/Touringcar/components/MapLayers/VehicleHeightsLayer.tsx

```diff
@@ -43,14 +43,8 @@ export const VehicleHeightsLayer = () => {
     })
   }
 
-  if (isError && error instanceof Error) {
-    console.error(error.message)
-  }
-
-  if (isLoading || !data) {
-    return null
-  }
-
+  if (isError && error instanceof Error) console.error(error.message)
+  if (isLoading || !data) return null
```

Comment:
For the record;p. I think we should use { } to surround the if blocks to protect against mistakes later.

Reply:
Recorded. 😙